### ISSUE TITLE
Incompatible with create-react-app

### ISSUE THREAD

**Issue by jimgswang** [suguru03/aigle#21]
This library is incompatible with [create-react-app](https://github.com/facebookincubator/create-react-app) since it uses UglifyJS2 in it's toolchain, which does not support the destructuring in imports.  

Ref https://github.com/terinjokes/gulp-uglify/issues/66

When we try to build our app, we receive
```
Name expected [./~/aigle/lib/internal/collection.js:3,0][static/js/main.7201b8fd.js:151,6]
```

create-react-app is the official recommended bootstrap project to create React apps. It is recommended to make this library make this library compatible so we can reach more users.

**Comment by suguru03** [comment]
Thank you for the issue, I'll fix it soon.

**Comment by suguru03** [comment]
@jimgswang I fixed it and published `v1.8.1`. Could you check it? :)

**Comment by jimgswang** [comment]
It works
